ospfd/ospf_vty.c

```diff
@@ -2223,6 +2223,9 @@ static void ospf_table_reinstall_routes(struct ospf *ospf,
 {
 	struct route_node *rn;
 
+	if (!rt)
```

Comment:
Can't we do this inside `route_top()`?

Reply:
All functions in `table.c` expect to receive a non-NULL pointer to table.
I'd prefer to left it as-is to keep the API consistent.